src/adapter.js

```diff
@@ -4,6 +4,24 @@ import $RefParser from 'json-schema-ref-parser';
 import yaml from 'js-yaml';
 import yamlAst from 'yaml-js';
 
+// These describe the type of annotations that are produced by this parser
+// and assigns a unique code to each one. Downstream applications can use this
+// code to group similar types of annotations together.
+const ANNOTATIONS = {
+  CANNOT_PARSE: {
+    type: 'error',
+    code: 1,
```

Comment:
Per my comment about using hyperlinks for this, `type` could be a link relation and `code` could be replaced by a URL.